sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/EventProcessorClientBuilderTest.java

```diff
@@ -51,7 +51,7 @@ public void testEventProcessorBuilderMissingProperties() {
                         + "sequence number of event = " + eventContext.getEventData().getSequenceNumber());
                 })
                 .processError(errorContext -> {
-                    System.out.printf("Error occurred in partition processor for partition {}, {}",
+                    System.out.printf("Error occurred in partition processor for partition %s, %s",
```

Comment:
Should we make it so that whenever we call `printf` we add a new line at the end?

Reply:
Yes, this should have a `%n` at the end